packages/@tinacms/api-git/src/server.ts

```diff
@@ -27,6 +27,7 @@ export interface GitServerConfig extends GitRouterConfig {
   pathToRepo: string
   pathToContent: string
   gitRemote?: string
+  maxBuildTime?: string
```

Comment:
The server doesn't actually use this variable, so it shouldn't expect to receive it.

```suggestion
```